### PR TITLE
Submit and get from api

### PR DESCRIPTION
**Pull request by Eldonratzlaff**
Create js files for api application
Submit and get from api

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
...
+
+const refreshbtn = document.getElementById('refresh-btn');
+refreshbtn.addEventListener('click', refreshEvent);
+
+const submitBtn = document.getElementById('submit-btn');
+submitBtn.addEventListener('click', submitEven);
+
+window.onload = () => {
+  refresh();
+};
```
Comment (omar-muhamad) [inline src/index.js:13]: - [x] At page load, There are some errors appear in the console, Please fix them. 

image

File: src/modules/submit.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const submitEven = async (e) => {
+  e.preventDefault();
+  const form = document.getElementById('form');
+  const name = form.elements.name.value;
+  const score = form.elements.score.value;
+  await submitScore(name, score);
+  refresh();
+  form.reset();
+};
```
Comment (omar-muhamad) [inline src/modules/submit.js:12]: - [ ] While trying to submit a new score to the leaderboard, there is an error happen (see the image below), Please fix that.

image

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Eldonratzlaff,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please add a command to run the webpack dev server to the scrips in the `package.json` file because now the is no command to run it.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mikethreels — APPROVED** [review]
Hi Eldon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**